### DISCUSSION TITLE
How do change the base style for ModalCloseButton?

### DISCUSSION THREAD

**Discussion by neilnahid** [chakra-ui/chakra-ui#4462] [Q&A] [answered]
I want to change the base style of the ModalCloseButton. I don't want to apply the style via variant.

```tsx
// I want this
<ModalCloseButton /> // assuming styling is same to "custom-style" via baseStyle

// not this ...
<ModalCloseButton variant="custom-style" />
```

**Comment by nikolovlazar** [comment, marked as answer]
You can override the default `Modal` style and add the styles you want to the `closeButton` part. The modal is a multipart component. You can check out its default style here: https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/modal.ts and how to override it here: https://chakra-ui.com/docs/theming/component-style#styling-multipart-components

**Reply by neilnahid** [reply]
Thank you so much! Silly of me to miss it in the documentation.
